kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -427,6 +429,7 @@
         metadataSidePanelContent: null,
         mobileSidePanelIsOpen: false,
         usingMeteredConnection: true,
+        isNetworkLibraryAvailable: false,
```

Comment:
I think maybe we could address part of @LianaHarris360's concern by defaulting this to `true`? Then once we've tried to load the network devices, the event you're emitting will then set this to false?